flexidate/__init__.py

```diff
@@ -219,11 +227,11 @@ def parse(self, date, **kwargs):
         # Parse the numbers intelligently
         # do not use std parser function as creates lots of default data
         res = dateutil_parser._parse(date, **kwargs)
-
+        res = res[0]
```

Comment:
On my local machine, tests are not passing for `python2.7`. But for passing in `travis`, I had to do this.
Fine for python3